### ISSUE TITLE
Add some useful text

### ISSUE THREAD

**Issue by birktj** [image-rs/image-rs.github.io#3]
Right now i feel like this site is kind of useless, we should at least add some paragraph with some descriptive text. A link to the blog would also be nice. And the "View on GitHub" button in the upper right corner should probably point to `github.com/image-rs` and not `github.com/image-rs/image-rs.github.io`.

**Comment by 197g** [comment]
Yup, I have some more 'empty' boxes for all our projects already but not pushed (the setup is there already). I'm planning to have a page for every project to fill with some nice example or results to promote them. Not sure where to put the link to the blog, probably above the project list.
